ui-speedspacechart/src/components/SpeedSpaceChart.tsx

```diff
@@ -145,6 +146,7 @@ const SpeedSpaceChart = ({
             color={backgroundColor}
             store={store}
             setStore={setStore}
+            setShowReticleLayer={setShowReticleLayer}
```

Comment:
that's an interesting point, the fact is I thought that we should inspect the graph even if settings panel is open, but not if we hover it, but if @thibautsailly agree with your proposition I m not against it

Reply:
ah if the panel is open it would hide the reticle even while we’re above the graph, I didn’t see that.

the behavior is good, no need to change that.

in that case you can just rename `showReticleLayer` into `mouseIsOverSettingsPanel` (that’s really what it’s testing) to avoid having what looks like reticle logic inside the settings component

Reply:
done